packages/angular/src/index.ts

```diff
@@ -0,0 +1,31 @@
+import { addGlobalEventProcessor, SDK_VERSION } from '@sentry/browser';
+
+if (addGlobalEventProcessor) {
+  addGlobalEventProcessor(event => {
```

Comment:
Can we not make this a global side effect? If we have to, we'll have to adjust the `sideEffects` key in `package.json`

Reply:
Fixed